content/docs/pipelines/tutorials/sdk-examples.md

```diff
@@ -1,35 +1,44 @@
 +++
 title = "Using the Kubeflow Pipelines SDK"
-description = "How to use the Kubeflow Pipelines API to upload a local file to create a new pipeline version"
-weight = 1
+description = "Examples that demonstrate how to use the Kubeflow Pipelines SDK"
```

Comment:
I wonder if we should remove the words "Examples that demonstrate" and instead just start with "How to...". The reason is that before I read this page, I couldn't figure out how it would differ from the page called **Experiment with the Pipelines Samples**. To me as a reader, there's not a clear distinction between examples and samples.

Reply:
It also makes the description shorter too, done.